src/test/java/KwidoPageTest.java

```diff
@@ -1,84 +1,80 @@
             // fill the last name filed
-//        Random rand = new Random();
-//        int lastNameNumber = rand.nextInt(100);
             lastName.sendKeys("Junior");
             // find the business name filed
-            WebElement businessName = ((ChromeDriver) driver).findElementByCssSelector("input[formcontrolname='businessName']");
+            WebElement businessName = driver.findElement(By.cssSelector("input[formcontrolname='businessName']"));
             // fill the business name filed
             businessName.sendKeys("Testpro");
             // find the phone number filed
-            WebElement phoneNumber = ((ChromeDriver) driver).findElementByCssSelector("input[formcontrolname='phone']");
+            WebElement phoneNumber = driver.findElement(By.cssSelector("input[formcontrolname='phone']"));
             // fill the phone name filed
             phoneNumber.sendKeys("858-402-5910");
             // find the email filed
-            WebElement email = ((ChromeDriver) driver).findElementByCssSelector("input[formcontrolname='email']");
-            // fill the email filed
-            //generate the timestamp
+            WebElement email = driver.findElement(By.cssSelector("input[formcontrolname='email']"));
+            // fill the email filed //generate the timestamp
             Calendar cal = Calendar.getInstance();
             SimpleDateFormat timeOnly = new SimpleDateFormat("HH-mm-ss");
             email.sendKeys(timeOnly.format(cal.getTime()) + "@gmail.com");
             // find the password filed
-            WebElement password = ((ChromeDriver) driver).findElementByCssSelector("input[formcontrolname='password']");
-            // turn off the password display and fill the password filed
+            WebElement password = driver.findElement(By.cssSelector("input[formcontrolname='password']"));
             password.sendKeys("passowrd123");
             // find the click accept the terms
-            WebElement clickTerms = ((ChromeDriver) driver).findElementByCssSelector(".col-lg-12 .ng-untouched");
+            WebElement clickTerms = driver.findElement(By.cssSelector(".col-lg-12 .ng-untouched"));
             // click the terms filed
             clickTerms.click();
             // find submit button
-            WebElement submitBtn = ((ChromeDriver) driver).findElementByCssSelector("button[type='submit']");
+            WebElement submitBtn = driver.findElement(By.cssSelector("button[type='submit']"));
             // click the submit button
             submitBtn.click();
-            //Thread.sleep(4000);
             WebDriverWait wait = new WebDriverWait(driver,15);
-            //wait.until(ExpectedConditions.visibilityOfAllElementsLocatedBy(element));
+            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".card-content .card-title")));
+            WebElement result = driver.findElement(By.cssSelector(".card-content .card-title"));
```

Comment:
"wait.until()" method return WebElement so you can use it for initializing the variable